src/pages/components/colorpickers.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Color picker
+description: >
+  The color picker component allows users to choose a color using a variety of input methods.
+package: '@zendeskgarden/react-colorpickers'
+components:
+  - colorpickers/src/elements/Colorpicker/index.tsx
+  - colorpickers/src/elements/ColorpickerDialog/index.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To let users change or update brand colors</li>
+      <li>To let users configure chart colors</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Color picker
+
+Color picker allows users to choose colors using sliders and input fields.
+
+import ColorpickerDefault from '../../examples/colorpickers/ColorpickerDefault.tsx';
+import ColorpickerDefaultCode from '!!raw-loader!../../examples/colorpickers/ColorpickerDefault.tsx';
+
+<CodeExample code={ColorpickerDefaultCode}>
+  <ColorpickerDefault />
+</CodeExample>
+
+### Color picker dialog
```

Comment:
Use `Dialog` for alignment with other component pages where we rarely use the heading to refer back to the API component name.